NUEntity.js

```diff
@@ -29,7 +29,7 @@ export default class NUEntity extends NUObject {
             isEditable: false }),
         lastUpdatedDate: new NUAttribute({
             localName: 'lastUpdatedDate',
-            attributeType: NUAttribute.ATTR_TYPE_NUMBER,
+            attributeType: NUAttribute.ATTR_TYPE_STRING,
```

Comment:
This too should be set only by the server. Let's make this read only in UI

Reply:
ok